packages/web-components/src/components/ic-card/ic-card.stories.mdx

```diff
@@ -836,3 +836,183 @@ import Button from "../ic-button/readme.md";
     ></a>`}
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  clickable: false,
+  disabled: false,
+  expandable: false,
+  fullWidth: false,
+  heading: "Heading",
+  href: "/",
+  message: "Message",
+  subheading: "Subheading",
+  iconSlot: true,
+  badgeSlot: true,
+  imageTopSlot: true,
```

Comment:
<img width="1107" alt="Screenshot 2024-06-19 at 13 38 02" src="https://github.com/mi6/ic-ui-kit/assets/113986285/11ae8cf6-249e-4926-924b-1449a5ffa1c4">

Not sure if conditional rendering in the story or the actual code as the padding / margin for the empty slots can still be seen in the card

Reply:
I think this is a storybook nuance because on the website where no image-top is used, there isn't extra padding. It's probably to do with toggling the slot between "image-top" and "" that's messing it up somewhere, but I haven't found another way to display this other than having a separate playground example for each slot.